### PR TITLE
NodeJS WebSocket Support

### PR DESCRIPTION
**Pull request by marinofranz**
This pull request implements #3 
This allows the module to be used in the browser & NodeJS

### CODE REVIEW

File: src/classes/Funk.ts [hunk 1]
```diff
@@ -25,9 +26,15 @@ export class Funk {
      */
     constructor(options?: FunkOptions) {
         if (options) this.options = options;
-        this.ws = new WebSocket(this.options.urlOverride || "wss://funk.statio.cc");
-        this.ws.onopen = () => this.onOpen(this);
-        this.ws.onmessage = (data: string | MessageEvent<any>) => this.onMessage(this, data);
+        if (typeof process === "object") {
```
Comment (OfficialCRUGG) [inline src/classes/Funk.ts:29]: This method of checking the environment is not completely reliable. Instead, check if `typeof window === 'undefined'` for a more reliable way to check.

**Review by OfficialCRUGG — CHANGES_REQUESTED** [review]
Please implement the requested change. Otherwise, everything looks good to me!

File: src/classes/Funk.ts (hunk 1 above)
Comment (marinofranz) [reply src/classes/Funk.ts]: That's fixed :)

**Review by OfficialCRUGG — APPROVED** [review]
lgtm! 👍

**Review by OfficialCRUGG — APPROVED** [review]
lgtm

**Pull request merged**